tests/unit/test_nc_merge.py

```diff
@@ -186,6 +186,7 @@ def test_same_data_type(self):
         output_data_type = out_dataset[test_variable].datatype
         self.assertEqual(input_data_type, output_data_type, 'Should be equal')
 
+    @skip("Test disabled for DAS-2216 quick fix that ignores missing reprojections")
```

Comment:
What broke this test (also sorry I'm running out of time)

Reply:
Looks like this has been fixed by using `ExcludedScienceVariables`. I'm going to resolve this conversation thread.